tests/wrappers.py

```diff
@@ -0,0 +1,24 @@
+import gym
+
+
+class AutoStopEnv(gym.Wrapper):
+    """A env wrapper that stops rollout at step max_path_length."""
+
+    def __init__(self, env=None, env_name="", max_path_length=100):
+        if env_name:
+            super().__init__(gym.make(env_name))
```

Comment:
Why do we need the env_name? Why can't we just pass the environment?

Reply:
You can do both.

Reply:
I was referring to [this](https://github.com/rlworkgroup/garage/blob/master/garage/envs/base.py#L44).

Reply:
I think Anson was asking why you didn't use the standard `gym.Wrapper` interface [here](https://github.com/openai/gym/blob/master/gym/core.py#L209)

I also think that `gym.Wrapper` is preferable, but it's test code so I won't insist on any changes.

Reply:
Thanks for pointing it out. Yes I agree that one is better. I didn't pay much attention to this class because it was implemented before and I just pasted it here. My alter goal is that we can remove this when the new sampler is finished. Then this wrapper won't be required.

Reply:
then it doesn't seem we need to talk much about it, anyway :)